util/util.go

```diff
@@ -625,7 +625,7 @@ func RemountVolume(volumeName string) error {
 	}
 
 	// The record schema is like: `<Device Path> <Mount Point> <Mount Options>`
-	res, err := nsExec.Execute("bash", []string{"-c", "mount | grep -v grep | grep ext4 | awk '{print $1,$3,$6}'"})
+	res, err := nsExec.Execute("bash", []string{"-c", "mount | grep ext4 | awk '{print $1,$3,$6}'"})
```

Comment:
why remove `grep -v grep`?

Reply:
The flag `-v` means excluding the string. But the output lines of `mount` won't contain the substring `grep` hence `grep -v grep` is redundant.